compiler/rustc_metadata/src/rmeta/decoder.rs

```diff
@@ -875,11 +875,9 @@ impl MetadataBlob {
                         let def_name = if item == CRATE_DEF_INDEX {
                             rustc_span::symbol::kw::Crate
                         } else {
-                            def_key
-                                .disambiguated_data
-                                .data
-                                .get_opt_name()
-                                .unwrap_or_else(|| Symbol::intern("???"))
+                            def_key.disambiguated_data.data.get_opt_name().unwrap_or(
+                                rustc_span::sym::question_mark_question_mark_question_mark,
```

Comment:
Suggestion: `unknown_def_name`? Or don't convert this instance because this is in printing code isn't it?